libs/daohaus-connect-feature/src/components/ConnectButton/UserConnetedDropdown.tsx

```diff
@@ -54,9 +56,7 @@ export const UserConnectedDropdown = ({ isSm }: { isSm: boolean }) => {
                 </ParMd>
               )}
               <ParXs color={theme.button.primary.text}>
-                {chainId && validNetwork
-                  ? `@${getNetworkName(chainId)}`
-                  : 'Wrong Network'}
+                {(networkName && `@${networkName}`) || 'Unknown'}
```

Comment:
Wouldn't 'Wrong Network' communicate the point more effectively to the user than 'Unknown"?